exporter/stackdriverexporter/observability.go

```diff
@@ -45,7 +38,22 @@ var viewPointCount = &view.View{
 	TagKeys:     []tag.Key{statusKey},
 }
 
-func recordPointCount(ctx context.Context, points int, status string) {
+func recordPointCount(ctx context.Context, success, dropped int, grpcErr error) {
+	if success > 0 {
+		recordPointCountDataPoint(ctx, success, "OK")
+	}
+
+	if dropped > 0 || grpcErr != nil {
+		var st string
+		s, ok := status.FromError(grpcErr)
+		if ok {
+			st = statusCodeToString(s)
+		}
+		recordPointCountDataPoint(ctx, dropped, st)
```

Comment:
if `grpcErr == nil` (which seems possible based on your condition above), `st` will be an empty string.
In that case you will report on dropped data points with empty status. Is that intended?

Reply:
Similarly, if `dropped == 0` but `grpcErr != nil`, you will record 0 dropped data points here.
Maybe you should change the condition to
```go
if dropped > 0 && grpcErr != nil
```
or just 
```go
if dropped > 0
```
depending on intended behaviour.

Reply:
For the status, I just wanted to replicate what the OC gRPC plugin did: https://github.com/census-instrumentation/opencensus-go/blob/f6172e422374225c6748bd2a9c872e88c2eb43d4/plugin/ocgrpc/stats_common.go#L135

Changed the condition to `if dropped > 0`